### PR TITLE
Prepare Identity to use December release version.

### PR DESCRIPTION
**Pull request by hemanttanwar**
December version update for identity library to 1.0.1

### CODE REVIEW

File: eng/jacoco-test-coverage/pom.xml [hunk 1]
```diff
@@ -80,7 +80,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-identity</artifactId>
-      <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-identity;current} -->
+      <version>1.0.1</version> <!-- {x-version-update;com.azure:azure-identity;current} -->
```
Comment (alzimmermsft) [inline eng/jacoco-test-coverage/pom.xml]: Is this meant to be a patch or minor version bump?

Comment (hemanttanwar) [reply eng/jacoco-test-coverage/pom.xml]: We spoke in standup yesterday  and we are good with 1.0.1 version.

**Pull request merged**